content/references/gui/index.md

```diff
@@ -0,0 +1,11 @@
+---
+kind: reference
+sidebar: reference-gui
+next: /references/gui/kata-editor/
+---
+
+# Codewars GUI reference
```

Comment:
I don't think we say GUI for Web apps because it's implied. Sounds awkward to me.

Maybe just `/references/ui/` and UI reference? Or don't even have this subdirectory and tag them with `ui`.

Importing it as is and improving later is fine.

Reply:
> I don't think we say GUI for Web apps because it's implied. Sounds awkward to me.

Okay, it's probably language thing that it sounds ok to me. I will change GUI to UI.

> Or don't even have this subdirectory and tag them with `ui`.

I divided it into folders to keep future organization easier, because I expect to have many screenshots around here, so they will have their proper place instead of piling up in one dir.

Reply:
I've replaced GUI with UI. If you have no more remarks, I will merge it.